src/xrGame/level_script.cpp

```diff
@@ -685,6 +708,8 @@ IC static void CLevel_Export(lua_State* luaState)
         def("spawn_item", &spawn_section),
         def("get_active_cam", &get_active_cam),
         def("set_active_cam", &set_active_cam),
+        def("get_start_time", &get_start_time),
```

Comment:
Let's use lambdas for all new exported functions, like this:
```suggestion
        def("get_start_time", +[]() { return xrTime(Level().GetStartGameTime()); }),
```

Reply:
Fixed